src/components/learner-credit-management/search/CatalogSearch.jsx

```diff
@@ -1,24 +1,25 @@
 import React from 'react';
 import algoliasearch from 'algoliasearch/lite';
 import { Configure, InstantSearch } from 'react-instantsearch-dom';
-import PropTypes from 'prop-types';
 
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
 import { SearchHeader } from '@edx/frontend-enterprise-catalog-search';
 
 import { configuration } from '../../../config';
 import CatalogSearchResults from './CatalogSearchResults';
-import { SEARCH_RESULT_PAGE_SIZE } from '../data';
+import { SEARCH_RESULT_PAGE_SIZE, useBudgetId, useSubsidyAccessPolicy } from '../data';
 
-const CatalogSearch = ({ catalogUuid }) => {
+const CatalogSearch = () => {
   const searchClient = algoliasearch(configuration.ALGOLIA.APP_ID, configuration.ALGOLIA.SEARCH_API_KEY);
-  const searchFilters = `enterprise_catalog_uuids:${catalogUuid} AND learning_type:'course'`;
+  const { subsidyAccessPolicyId } = useBudgetId();
+  const { data: subsidyAccessPolicy } = useSubsidyAccessPolicy(subsidyAccessPolicyId);
+  const searchFilters = `enterprise_catalog_uuids:${subsidyAccessPolicy.catalogUuid} AND learning_type:'course'`;
```

Comment:
[sanity check] [This PR ](https://github.com/openedx/frontend-app-admin-portal/pull/1065/files#diff-20577a671cdee640bb2fd44b7d7b7cc000afb0215773a4709b5811ad39cc3d80R19) correctly has the search filters based on the `content_type:'course'`, but here it's listed as `learning_type:'course'`.

Given we want all course types (which may include multiple learning types), we should be filtering based on `content_type:'course'`.

Reply:
This is fixed after rebasing!